packages/gql/codegen.ts

```diff
@@ -111,6 +112,27 @@ export const SUBGRAPHS = {
       return this.endpoints()[network];
     },
   },
+  [Subgraph.PoolSnapshots]: {
```

Comment:
we already have this subgraph here, so you don't need to add it again. You can just add a new query to the existing balancer subgraph